tests/Data/Entries/EntryTest.php

```diff
@@ -74,9 +74,17 @@ public function it_sets_and_gets_the_slug()
     /** @test */
     public function the_slug_gets_slugified()
     {
+        Facades\Site::setConfig(['default' => 'en', 'sites' => [
+            'en' => ['locale' => 'en_US', 'url' => '/'],
+            'da' => ['locale' => 'da_DA', 'url' => '/da/'],
+        ]]);
+
         $entry = new Entry;
-        $entry->slug('foo bar');
-        $this->assertEquals('foo-bar', $entry->slug());
+        $entry->slug('foo bar æøå');
```

Comment:
`foo bar & æøå`

Reply:
Ampersands weren't handled before this, they were just stripped out. I'll leave this change for another PR.

Illuminate\Support\Str::slug() strips them out.

Reply:
Ahhh okay. They aren't stripped out through `slugify` though, that's why I thought it was the same for PHP.
<img width="945" alt="image" src="https://user-images.githubusercontent.com/1186125/158255979-c762350f-0cd5-40ce-b960-03be4ae6186e.png">

Reply:
Oh Javascript. I see. 🤔  Okay then we can work something out in a separate PR/issue. Ideally I'd like the JavaScript to mimic the PHP.